pkg/ddc/vineyard/operations/base_test.go

```diff
@@ -19,32 +19,33 @@ import (
 	"net/http"
 	"reflect"
 	"testing"
+	"time"
 
-	"github.com/brahma-adshonor/gohook"
+	. "github.com/agiledragon/gomonkey/v2"
 	"github.com/go-logr/logr"
 )
 
-func mockHttpGet(url string) (*http.Response, error) {
-	if url == "http://vineyard-0.vineyard-svc.default.svc.cluster.local:8080/metrics" {
-		return &http.Response{
-			StatusCode: http.StatusOK,
-			Body:       ioutil.NopCloser(bytes.NewBufferString("metric for pod 1")),
-		}, nil
-	}
-	if url == "http://vineyard-1.vineyard-svc.default.svc.cluster.local:8080/metrics" {
-		return &http.Response{
-			StatusCode: http.StatusOK,
-			Body:       ioutil.NopCloser(bytes.NewBufferString("metric for pod 2")),
-		}, nil
-	}
-	return nil, nil
-}
-
 func TestVineyardFileUtils_ReportSummary(t *testing.T) {
-	err := gohook.Hook(http.Get, mockHttpGet, nil)
-	if err != nil {
-		t.Fatalf("could not hook http.Get: %v", err)
+	client := &http.Client{
+		Timeout: 20 * time.Second,
 	}
+	patch := ApplyMethod(reflect.TypeOf(client), "Get",
+		func(_ *http.Client, url string) (resp *http.Response, err error) {
+			if url == "http://vineyard-0.vineyard-svc.default.svc:8080/metrics" {
+				return &http.Response{
```

Comment:
I don't suggest adding `-svc` to the service name. Can you remove it from the chart. Thanks.

Reply:
Ok, I will submit a PR to address it.